.github/workflows/release.yml

```diff
@@ -3,7 +3,7 @@ name: Release
 on:
   push:
     branches:
-      - main
+      - pypi
```

Comment:
Before merging in, change this back to main!

Reply:
![image](https://github.com/executablebooks/mystmd/assets/913249/aa8f5d9d-7716-472e-bfff-02feedb5b8be)